src/Main/ProjectFile/CompilerComponent.cpp

```diff
@@ -51,4 +51,10 @@ ErrorCode CompilerComponent::Load(QXmlStreamReader *reader) {
   return {};
 }
 
+void CompilerComponent::LoadDone() {
+  std::filesystem::path configJsonPath =
```

Comment:
If the file does not exist (Synthesis didn't execute yet), this should not be run. Test if file exist.